app/src/main/java/amazingme/activities/util/DialogHelper.java

```diff
@@ -0,0 +1,29 @@
+package amazingme.activities.util;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.content.DialogInterface;
+import android.os.Build;
+
+public class DialogHelper {
```

Comment:
@masteryoda123 what other dialog boxes are there?? 

my next push on the arch branch might abstract this a bit more to make  it a bit more flexible for later use. 

if you had a link you used for creating this (just like a template you used) i'd love that. 

otherwise ignore this. solid job

Reply:
There's a couple more types. I don't think this implementation here is the cleanest way either. But it should do for now, and we should clean up as we add more stuff to it.

https://developer.android.com/guide/topics/ui/dialogs.html

Reply:
got it saved. preciate it.